libs/blocks/merch-card/merch-offer-select.js

```diff
@@ -42,6 +49,18 @@ export const initOfferSelection = (merchCard, offerSelection, quantitySelector)
     merchOffers.append(createMerchOffer(option, quantitySelector));
   });
   merchOffers.querySelectorAll('a[is="checkout-link"]').forEach((link) => { link.setAttribute('slot', 'cta'); });
+  if (isHorizontal(merchOffers)) {
+    merchOffers.setAttribute('variant', 'horizontal');
+  }
+  merchOffers.querySelectorAll('merch-offer').forEach((offer) => {
+    const links = offer.querySelectorAll('a[is="checkout-link"]');
+    if (links.length > 1) {
+      links[0].setAttribute('slot', 'secondary-cta');
```

Comment:
I thought we didn't need an explicit slot named `secondary-cta`

the problem I see is that, authors can add a merch link as primary or secondary CTA in the order they want, but here it is assumed that the first link is secondary, and the second links is primary.

I think we should be less opinionated and let authors stack CTAs in the order they want.

Reply:
can this happen?

Reply:
I think yes if my understanding of the above code is correct.

Reply:
well i guess i can indeed "just" dub the links in the order they came, how they came under the cta "slot"

Reply:
i agree with you, but for now it's complicated to change this as target placeholder/slot is supposed to be a link. As discussed offline we should quickly make this better, and have that solution in the meanwhile

Reply:
as discussed, we'll work on related technical debt (i'll attach it to the actual ticket) asap, but this is complicated to change now